app/models/test_track/application_identity.rb

```diff
@@ -24,8 +24,6 @@ def initialize(app_name)
       @app_name = app_name
     end
 
-    private
-
```

Comment:
This should no longer be necessary if [this change](https://github.com/Betterment/test_track_rails_client/pull/132/files#r1688949530) is accepted.